modules/alloy/src/main/ts/ephox/alloy/behaviour/invalidating/InvalidateApis.ts

```diff
@@ -46,13 +46,7 @@ const markInvalid = (component: AlloyComponent, invalidConfig: InvalidatingConfi
   });
 };
 
-const query = (component: AlloyComponent, invalidConfig: InvalidatingConfig, _invalidState: Stateless): Future<Result<any, string>> => {
-  return invalidConfig.validator.fold(() => {
-    return Future.pure(Result.value(true));
-  }, (validatorInfo) => {
-    return validatorInfo.validate(component);
-  });
-};
+const query = (component: AlloyComponent, invalidConfig: InvalidatingConfig, _invalidState: Stateless): Future<Result<any, string>> => invalidConfig.validator.fold(() => Future.pure(Result.value(true)), (validatorInfo) => validatorInfo.validate(component));
```

Comment:
Another one where it's all been shoved onto a single line.

Reply:
Fixed